server/src/thirdparty/__tests__/services/github.test.js

```diff
@@ -0,0 +1,34 @@
+import request from 'supertest';
+import app from '../../../app';
+
+describe('Should return services and actions accordingly', () => {
+  test('should return the list of services', async () => {
+    const res = await request(app)
+      .get('/services')
+      .send();
+
+    expect(res.statusCode).toEqual(200);
+    expect(res.body).toHaveProperty('service');
+    expect(res.body.service).toStrictEqual(['github']);
+  });
+
+  test('should return the actions for github service', async () => {
+    const res = await request(app)
+      .get('/actions/github')
+      .send();
+
+    expect(res.statusCode).toEqual(200);
+    expect(res.body).toHaveProperty('actions');
+    expect(res.body.actions).toHaveLength(1);
+  });
+
+  test('should return the triggers for github service', async () => {
+    const res = await request(app)
+      .get('/triggers/github')
```

Comment:
For slack, it is /slack/event. so it will be difficult for client to use different apis for different apps.